packages/optimizers/svgo/src/SVGOOptimizer.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+
+import {Optimizer} from '@parcel/plugin';
+import ThrowableDiagnostic from '@parcel/diagnostic';
+import {blobToString} from '@parcel/utils';
+
+import * as svgo from 'svgo';
+import path from 'path';
+
+export default (new Optimizer({
+  async loadConfig({config}) {
+    let configFile = await config.getConfig(
+      [
+        '.svgorc',
+        '.svgorc.json',
+        '.svgorc.yaml',
+        '.svgorc.yml',
+        'svgo.config.js',
+        '.svgo.yml',
```

Comment:
The only config files read by svgo 2 itself is `svgo.config.js`, so maybe we should only support `svgo.config.js(on)`